commit 047b72eb0d30d5cc9ebd8af238f174e3dc45a8b7 (refs/changes/54/4854/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-09-24T09:20:56-07:00 (1 year ago)
    
    TRITON-782 napi sdc-client test failure in nightly-2: "invalid MAC address"
    Reviewed by: Pedro Palazón Candel <pedro@joyent.com>
    Approved by: Pedro Palazón Candel <pedro@joyent.com>

diff --git a/test/napi.test.js b/test/napi.test.js
index 84e3c65..4d061a8 100644
--- a/test/napi.test.js
+++ b/test/napi.test.js
@@ -31,7 +31,7 @@ function pseudoRandomMac() {
         var diff = maxVal - minVal + 1.0,
             val = Math.random() * diff;
         val += minVal;
-        return Math.round(val);
+        return Math.floor(val);
     }
     mac[3] = randomInt(0x00, 0x7f);
     mac[4] = randomInt(0x00, 0xff);
